src/screens/WalletConnectRedirectSheet.js

```diff
@@ -27,12 +28,12 @@ const emojisMap = {
 };
 
 const titlesMap = {
-  'connect': "You're connected!",
-  'reject': 'Connection canceled',
-  'sign': 'Message signed!',
-  'sign-canceled': 'Transaction canceled!',
-  'transaction': 'Transaction sent!',
-  'transaction-canceled': 'Transaction canceled!',
+  'connect': lang.t('walletconnect.titles.connect'),
+  'reject': lang.t('walletconnect.titles.reject'),
+  'sign': lang.t('walletconnect.titles.sign'),
+  'sign-canceled': lang.t('walletconnect.titles.sign_canceled'),
+  'transaction': lang.t('walletconnect.titles.transaction'),
```

Comment:
would be nice to rename this to `walletconnect.titles.transaction_sent`

Reply:
@jinchung Updated, thanks!